content/en/agent/logs/_index.md

```diff
@@ -163,7 +162,7 @@ List of all available parameters for log collection:
 | `port`           | Yes      | If `type` is **tcp** or **udp**, set the port for listening to logs.                                                                                                                                                                                                                                                                                    |
 | `path`           | Yes      | If `type` is **file** or **journald**, set the file path for gathering logs.                                                                                                                                                                                                                                                                            |
 | `channel_path`   | Yes      | If `type` is **windows_event**, list the Windows event channels for collecting logs.                                                                                                                                                                                                                                                                    |
-| `service`        | Yes      | The name of the service owning the log. If you instrumented your service with [Datadog APM][10], this must be the same service name.                                                                                                                                                                                                                     |
+| `service`        | Yes      | The name of the service owning the log. If you instrumented your service with [Datadog APM][9], this must be the same service name. You can inject `service` with the APM tracer or, if you're using Kubernetes or Docker, use labels to set your service in logs. It recommended to use [unified service tagging][10] when configuring `service`.                                                                                                                                                                                                                      |
```

Comment:
I would recommend to make it simpler. We should not explain here that it can be done with labels or injections or other methods if more are added in the future but simply link to the unified service tagging main page which should explain how to do this.

```suggestion
| `service`        | Yes      | The name of the service owning the log. If you instrumented your service with [Datadog APM][9], this must be the same service name. Check the [unified service tagging][10] instructions when configuring `service` across multiple data types.                                                                                                                                                                                                                      |
```